glue/sales_val_flagging.py

```diff
@@ -701,7 +727,7 @@ def write_to_table(df, table_name, s3_warehouse_bucket_path, run_id):
     NA_Dates AS (
         SELECT
             MIN(DATE_TRUNC('MONTH', sale.sale_date)) - INTERVAL '{rolling_window_num_sql}' MONTH AS StartDate,
-            MAX(DATE_TRUNC('MONTH', sale.sale_date)) AS EndDate
+            MAX(date_add('day', -1, date_add('month', 1, DATE_TRUNC('MONTH', sale.sale_date)))) AS EndDate
```

Comment:
For our most recent sales data, it was clipping the ingest on the first of that month. This change makes sure we get all of the data from that month.